src/pages/return/components/buyer-address-fields/BuyerAddressFields.js

```diff
@@ -0,0 +1,167 @@
+import { toast } from 'react-toastify';
```

Comment:
if this components are same as the add order flow then you may move these components to common or somewhere and reuse them it will be good to centralize UI components.